gslib/commands/stat.py

```diff
@@ -53,10 +53,7 @@
     gsutil ls -L gs://some-bucket/some-object
 
   but is more efficient because it avoids performing bucket listings and gets
-  the minimum necessary amount of object metadata. Moreover, because it avoids
-  performing bucket listings (which are eventually consistent) the gsutil stat
```

Comment:
Why did you choose to remove this section?  IMO, it seems useful.  The only thing I'd change here is that bucket and object listings *may* be eventually consistent, depending on the storage provider (e.g. listing objects in an S3 bucket).

Reply:
I hadn't considered non-Google Cloud Storage providers. Edited.